parquet-column/src/main/java/org/apache/parquet/column/values/bloomfilter/BlockSplitBloomFilter.java

```diff
@@ -328,6 +329,20 @@ public long hash(Object value) {
     return doHash();
   }
 
+  @Override
+  public boolean equals(Object object) {
+    if (object == this) {
+      return true;
+    }
+    if (object instanceof BlockSplitBloomFilter) {
+      BlockSplitBloomFilter that = (BlockSplitBloomFilter) object;
+      return Arrays.equals(this.bitset, that.bitset)
```

Comment:
Why only these 3 checks while there are several other properties like minimumBytes?

Reply:
@shangxinli
Sorry for the late reply.
Other properties like `BYTES_PER_BLOCK` are for internal use, any small change will lead to a different `bitset`.
So I thought no need to check them.

Reply:
It depends on the usage of this method. BYTES_PER_BLOCK is fine but why minimumBytes is not checked since maximumBytes is chcked?

Reply:
The [implementation](https://github.com/apache/parquet-mr/blob/bloom-filter/parquet-column/src/main/java/org/apache/parquet/column/values/bloomfilter/BlockSplitBloomFilter.java) of `bloom-filter` branch takes in the constructor only `maximumBytes`. You re right, it's a mistake.   
What about ignoring both the `maximumBytes` and the `minimumBytes`, since they will lead to a different size of bitset?

Reply:
Again, it depends on the usage of this method. If it is a mistake, I would prefer to fix that mistake and come back with right approach.

Reply:
I’ll see what I can do!

Reply:
What about it this approach?

Reply:
LGTM